scripts/lint_site.sh

```diff
@@ -18,9 +18,21 @@ set -e
 
 FAILED=0
 
-if [[ "$#" -ne 0 ]]; then
+if [[ -n "${BRANCH}" ]]; then
+    files_changed="$(git diff --name-only --diff-filter=b "$(git merge-base HEAD "$BRANCH")")"
+    if (echo "${files_changed}" | grep -q /en/); then
+        LANGS+="en "
+    fi
+    if (echo "${files_changed}" | grep -q /zh/); then
+        LANGS+="zh "
+    fi
+    if (echo "${files_changed}" | grep -q /pt-br/); then
+        LANGS+="pt-br "
+    fi
+elif [[ "$#" -ne 0 ]]; then
     LANGS="$*"
 else
+    echo "if BRANCH is set to the source branch, ie. master, the linter will only check languages within the PR."
     LANGS="en zh"
```

Comment:
Maybe just fail in this case?
```suggestion
    echo "ERROR: no languages specified!"
    exit 1
```